### ISSUE TITLE
fly4free.pl

### ISSUE THREAD

**Issue by kulfoon** [easylistpolish/easylistpolish#28]
Eppty ad containers + ad remains / text "reklama" as per https://github.com/MajkiIT/polish-ads-filter/issues/10634

**Comment by easylistpolish** [comment]
Can't reproduce

**Comment by kulfoon** [comment]
Right, again that's Fanboy's Enhanced Tracking List triggering the ads, feel free to close it.
